src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FinalTransferConfirmationRouteSpec.scala

```diff
@@ -43,6 +60,16 @@ class FinalTransferConfirmationRouteSpec extends AnyFlatSpec with Matchers with
     checkFinalTransferConfirmationExists(consignmentId)
   }
 
+  it should "return all requested fields from inserted final judgment transfer confirmation consignment metadata properties" in {
```

Comment:
Could you change `it` to `"The api` It does work as you've got it but all the other tests are in the `"" should ""` format.